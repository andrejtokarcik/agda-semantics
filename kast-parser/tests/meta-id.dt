data Nat : Set 0 where
  Zero : Nat,
  Succ : Pi x Nat Nat

; expr = (Lambda a (Set 0) (Lambda x a x)) ? Zero
