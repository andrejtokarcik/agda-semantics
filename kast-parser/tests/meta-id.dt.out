<T>
    <k>
        .K
    </k>
    <mgu>
        subst(.KList)
    </mgu>
    <tenv>
        expr |-> Nat
    </tenv>
    <data>
        Nat |-> Set 0
        Succ |-> (Pi x Nat Nat)
        Zero |-> Nat
    </data>
    <fun>
        expr |-> (((Lambda a Set 0 (Lambda x a x)) #symExpr(2)) Zero)
    </fun>
    <meta>
        #symExpr(2) |-> ( Nat , Set 0 )
    </meta>
</T>
